apps/teams-test-app/e2e-test-data/chat.json

```diff
@@ -48,7 +48,7 @@
       "version": ">2.2.0 && <=2.17.0",
       "boxSelector": "#box_openChat2",
       "inputValue": {
-        "user": ["testUpn"],
+        "user": "testUpn",
```

Comment:
Instead of here, it looks the changes should be made in line 63 and 66?

where the input is "testUpn", and expectedAlertValue is [[\"testUpn\"]] instead of [\"testUpn\"]?

Reply:
```suggestion
        "user": ["testUpn"],
```